packages/dds/tree/src/test/feature-libraries/sequence-field/rebase.spec.ts

```diff
@@ -352,4 +377,62 @@ describe("SequenceField - Rebase", () => {
         ];
         assert.deepEqual(actual, expected);
     });
+
```

Comment:
It would be nice to have test cases for:
- A pair of inserts targetting interior gaps of a lengthy delete
- A pair of inserts where the later insert is meant to land before the earlier insert

Reply:
But those could be tackled in later PRs. Feel free to resolve this comment.